base_projects/Drivetrain/src/main/include/logging/SysIdLogger.h

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <sstream>
```

Comment:
These aren't used in this header; remove them. If they are used in SysIdLogger.cpp, they should be moved there.